spaceship.zsh

```diff
@@ -738,6 +746,25 @@ spaceship_julia() {
     "$SPACESHIP_JULIA_SUFFIX"
 }
 
+# JAVA
+# Show current Java version
+spaceship_java() {
+  [[ $SPACESHIP_JAVA_SHOW == false ]] && return
+
+  _exists java || return
+
+  # Show Java version only for Java-related folders
+  [[ -f pom.xml ]] || return
+
+  local java_version=$(java -version 2>&1 | head -n 1 | awk -F '"' '{print $2}')
```

Comment:
I think [`sdkman`](http://sdkman.io/) is fairly popular too. 
If we add support for that, we can get groovy/gradle/etc versions as well.